test/icc-file-parsing.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
```

Comment:
This was nice, using real ICC files.

Since the file tests `icc-tags.js`, should it be named `icc-tags-spec.js` or do you think this is something outside of the rest of the test structure?

Reply:
idk, loading full files is not really unit test any more but on the other hand it is fast enough to run in npm test watch mode and it increases confidence. I'd leave it here but that is mostly because I don't have a strong opinion :)

Reply:
Let's leave it. :-)